gulpfile.js

```diff
@@ -217,7 +217,6 @@ gulp.task("tslint", function () {
 });
 
 gulp.task("clean", function () {
-    var del = require("del");
```

Comment:
(this is unrelated to the device list parsing, but i noticed this was redeclared after already being declared at the top of the file so i got rid of this)